frontend/simple/js/state.js

```diff
@@ -57,6 +64,24 @@ const mutations = {
       mutations.addContract(state, contract)
     }
   },
+  deleteMail (state, hash) {
+    let mailbox = getMailbox(state)
+    if (mailbox) {
+      let index = state[ mailbox ].messages.findIndex(msg => msg.hash === hash)
+      if (index > -1) {
+        state[ mailbox ].messages.splice(index, 1)
+      }
+    }
+  },
+  readMail (state, hash) {
```

Comment:
This function name can also be improved. `readMail` doesn't really tell us what this function is really doing. We can make our code more self-documenting by being more specific, e.g.: `markMessageAsRead`.